src/main/java/com/yetanalytics/xapi/model/LangMap.java

```diff
@@ -44,31 +44,31 @@ public void put(String languageCode, String value) {
      * @param languageCode RFC 5646 Language Tag
      * @return The value in the language specified by the tag.
      */
-    public String get(String languageCode) {
+    public String get(LangTag languageCode) {
         return languageHashMap.get(languageCode);
     }
 
     /**
      * Remove an Entry from the Language Map
      * @param languageCode RFC 5646 Language Tag
      */
-    public void remove(String languageCode) {
+    public void remove(LangTag languageCode) {
```

Comment:
i wanted string overloads for everything.

Reply:
Note how in addition to adding the overloaded methods for both LangMaps and extensions, I also added the IFreeMap interface so that we can:
- enforce the contract of needing not just `get`, `put`, and `remove`, but overloaded variants of each
- have a single FreeMapSerializer instead of two separate serializers